allennlp/models/simple_tagger.py

```diff
@@ -42,10 +42,10 @@ def __init__(self, vocab: Vocabulary,
         self.tag_projection_layer = TimeDistributed(Linear(self.stacked_encoder.get_output_dim(),
                                                            self.num_classes))
 
-    # pylint: disable=arguments-differ
     def forward(self,  # type: ignore
```

Comment:
The tradeoff there is that then the logic has to be duplicated in every model that does this.  The idea with this class is essentially to _be_ that `get_embedding_representations()` method, with minimal requirements on the model it's wrapping.